### PR TITLE
try to not overwrite the global geojson and intercept exception to give a second chance 

### PR DESCRIPTION
**Pull request by fritruc**
try to not overwrite the global geojson
intercept exception to give a second chance

### CODE REVIEW

File: tools/osm-cadastre.py [hunk 1]
```diff
@@ -296,7 +296,17 @@ def count_sources(datatype, insee, force_download):
             );
             out tags qt;""".format(insee)
 
-    response = API.Get(request, responseformat='json', build=False)
+    for  retry in range(9, 0, -1):
+        try:
+            response = API.Get(request, responseformat='json', build=False)
+            break
+        except (overpass.errors.MultipleRequestsError, overpass.errors.ServerLoadError) as e:
+            import time
```
Comment (bagage) [inline tools/osm-cadastre.py]: please add `import` at the top of the file.

File: tools/osm-cadastre.py [hunk 2]
```diff
@@ -296,7 +296,17 @@ def count_sources(datatype, insee, force_download):
...
+            log.info("Oops : {} occurred. Will retry again {} times in a few seconds".format(type(e).__name__, retry ) )
```
Comment (bagage) [inline tools/osm-cadastre.py]: use `log.warning` instead?

File: tools/osm-cadastre.py [hunk 3]
```diff
@@ -296,7 +296,17 @@ def count_sources(datatype, insee, force_download):
...
+            if retry == 0:
+                log.info("No more retry left!")
```
Comment (bagage) [inline tools/osm-cadastre.py]: can remove this one (exception will be raised so no need for extra log).

**Comment by bagage** [comment]
Thanks for the PRs @fritruc! Both looks good, sound good to merge - just reviewed a few typos if you can fix them, or I'll do.

**Comment by fritruc** [comment]
done

**Pull request merged**